### PR TITLE
Updates Compute.Json to improve Documentation around PerformMaintenan… and ConvertToManagedDisks APIs

### PR DESCRIPTION
**Pull request by srvar**
…ce API

Updates Compute.Json to improve Documentation around PerformMaintenance API

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by hari-bodicherla — APPROVED** [review]

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json [hunk 1]
```diff
@@ -3109,7 +3109,7 @@
           "VirtualMachines"
         ],
         "operationId": "VirtualMachines_PerformMaintenance",
-        "description": "The operation to perform maintenance on a virtual machine.",
```
Comment (hyonholee) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: Could you also update the description of the previous versions?

**Review by lirenhe — APPROVED** [review]

**Comment by lirenhe** [comment]
@srvar , I found there is one active comment, please let me know if this PR is ready to merge.

**Comment by srvar** [comment]
> @srvar , I found there is one active comment, please let me know if this PR is ready to merge.

I have to update the PR with some other changes also. So not yet ready to merge.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json (hunk 1 above)
Comment (srvar) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: Which all versions would you like me to update?

**Review by smotwani — APPROVED** [review]

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json (hunk 1 above)
Comment (hyonholee) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json]: All previous versions and new version (compute.json is now updated to 2019-12-01)

**Review by hyonholee — CHANGES_REQUESTED** [review]
The version of compute.json is updated to 2019-12-01.  Please update new version and also all previous versions.

**Review by smotwani — APPROVED** [review]

**Pull request closed without merging**